Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -335,18 +335,20 @@ class PostRideSummaryViewController: PostRideViewController {
     
     @objc private func postRide() {
         //TODO: Networking Goes here - still needs to be debugged after backend fixes
-        guard let creatorID = currentRide.driver?.id else { return }
+        let creatorID = NetworkManager.shared.currentUser.id
         
         NetworkManager.shared.postRide(startID: currentRide.path.startLocationPlaceId, startName: currentRide.path.startLocationName, endID: currentRide.path.endLocationPlaceId, endName: currentRide.path.endLocationName, creator: creatorID, maxTravellers: currentRide.maxTravelers, minTravellers: currentRide.minTravelers, type: currentRide.type, isFlexible: currentRide.isFlexible, departureTime: currentRide.departureDatetime) { response in
             switch response {
             case .success(_):
                 self.dismiss(animated: true)
+                self.prevVC()
```

Comment:
Can you rename this function. It's unclear what it's doing and doesn't look very Swifty